apps/site/src/views/Schedule/sections/ClipboardSchedule/ClipboardSchedule.tsx

```diff
@@ -27,21 +27,32 @@ const dateTimeFormat = new Intl.DateTimeFormat("en", {
 	minute: "numeric",
 });
 
+const WEEKDAYS = [
+	"Sunday",
+	"Monday",
+	"Tuesday",
+	"Wednesday",
+	"Thursday",
+	"Friday",
+	"Saturday",
+];
+
```

Comment:
Suggestion: this approach restricts the localization options. We can create a new formatter for the day of the week, e.g.
```ts
const weekdayFormat = new Intl.DateTimeFormat("en", { weekday: "long" });
```